src/fetchtastic/utils.py

```
@@ -452,8 +497,24 @@ def make_github_api_request(
-    # Log API response info for debugging
-    logger.debug(f"GitHub API response: {response.status_code} for {url}")
+    # Track API request statistics and log first requests
+    global _api_request_count, _api_auth_used, _api_first_auth_logged, _api_first_unauth_logged
+    with _api_tracking_lock:
+        _api_request_count += 1
+        is_authenticated = bool(effective_token)
+        if is_authenticated:
+            _api_auth_used = True
+            # Log first authenticated request
+            if not _api_first_auth_logged:
+                logger.info(f"🔐 Making first authenticated GitHub API request")
```

Ruff (0.14.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)